admin/src/locales/en.json

```
@@ -1,6 +1,37 @@
 {
   "all_emails": "All users",
   "back": "back",
+  "contributionLink": {
+    "amount": "Amount",
+    "clear": "Clear",
+    "contributionLinks": "Contribution Links",
+    "create": "Create",
+    "cycle": "Cycle",
+    "endDate": "End-Date",
+    "maximumAmount": "Maximum amount",
+    "memo": "Memo",
+    "name": "Name",
+    "newContributionLink": "New Contribution Link",
+    "noContributionLinks": "No contribution links have been created.",
+    "noEndDate": "No end-date",
+    "noStartDate": "No start-date",
+    "options": {
+      "cycle": {
+        "daily": "daily",
+        "hourly": "hourly",
+        "monthly": "monthly",
+        "none": "no cycle",
+        "null": "Please select a cycle",
```

Came in my mind now:

```suggestion
        "null": "please select a cycle",
```

---

changed